### ISSUE TITLE
Power State Feature - Can't turn back on my DELL S2721

### ISSUE THREAD

**Issue by TrustKibou** [xanderfrangos/twinkle-tray#445]
Hey everyone! So I just discovered this (absolutely wonderful) application the other day and LOVE it... however, I was playing around with the settings earlier and decided to give the experimental "Power State" feature a try on one of my three desktop monitors. I turned it on in the Settings, clicked the icon in the system tray, clicked "Power Off", and voila, that monitor turned off.... however, I cannot get it to turn back on! I'm freaking out a tad... when I click the icon now, it gives me the option to "Power Off" the monitor again, but it doesn't actually do anything when I try clicking it. I've trying power cycling the monitor, unplugging the cables, both together, power cycling the PC itself... and nothing. I'm really worried at the moment. :( I shouldn't have even tried it, but it would have been such a handy feature!

I also found [these](https://linustechtips.com/topic/1389078-did-twinkle-tray-kill-my-monitor/) [two](https://github.com/xanderfrangos/twinkle-tray/issues/265) threads (both the same thread, just one posted on a different site and the other posted here) by someone with a similar issue, but different cause - I have tried resetting the monitor using the app (ControlMyMonitor) they suggested, but it's either not working or I don't know what I am doing. >_< Sorry for the newb nuisance. :c Thank you!

Also - the three monitors I use are all DELL S2721 and my tower is a pre-built. I can post those specs if they're needed!

**Comment by TrustKibou** [comment]
Okay, I am sorry for wasting anyone's time, but I actually fixed it thanks to continued Google searching - I unplugged the cables from the monitor and held the power button for 10 seconds, then plugged the cables back in and, voila, it turned back on. If you'd like me to delete this, I can! Or if you want to leave it up for anyone that may run into this in the future (unless I'll forever be the only one lol)

Thanks! And also thanks for developing such a helpful utility!
